dist/localization.js

```diff
@@ -0,0 +1,2 @@
+window.location.href = lang + '/index.html'; //redirect to the appropriate site
```

Comment:
It seems `window.location.replace` is better, a user will not be redirected back to the same page if the user uses the back button

Reply:
The back button seems to work as intended when I tested this. This localization script is only run once, when the user initially goes to the website so I don't see how the user would go to the same page.